Sender/generateTimestamps.cpp

```diff
@@ -0,0 +1,46 @@
+#include "generateTimestamps.h"
+
+RandomGenerator::RandomGenerator(double min, double max, unsigned long seed) :
+        _rand_seed(seed), _generator(seed), _min(min), _max(max), _distribution((min + max) / 2, (max - min) / 4) {}
+
+int RandomGenerator::operator ()()
+{
```

Comment:
() () is that a typo ?

Reply:
The first is overloading the operator, the second is passing no arguments to the overloaded ().

Reply:
ok I anticipated that, great.